test/README.md

```diff
@@ -73,6 +76,44 @@ $ pipenv install
 $ pipenv run script/cibuild
 ```
 
+## Onekey emulator
+
+### Dependencies
+
+In order to build the Onekey emulator, the [Nix](https://nixos.org) will need to be installed:
```

Comment:
Why is Nix necessary?

Reply:
The Nix is there to ensure build environment is reproducible, thus also ensures firmware build by user and official release matches (the binary part, not the header which contain signatures). As side effect, emulator has to be built with in Nix environment.

Nix also used in Trezor build process ([here](https://github.com/trezor/trezor-firmware/blob/main/ci/shell.nix))

Reply:
So, is Nix a must? Aren't there other ways to do it?

Reply:
HWI CI test no longer require Nix